City-of-Pori/pori-kada
# PORI-575: Facts liftup styles

Steps to test:
1. Go to https://businesspori.dev.wunder.io/fakta-fact-testi
2. Add new facts (create new content -> Fact -> attach to Fakta-fact-testi page) and see that the frames and colors are added correctly
2. See that the facts liftup looks according to designs
